pkg/manifest/testdata/simple.porter.yaml

```diff
@@ -4,7 +4,8 @@ mixins:
 name: hello
 description: "An example Porter configuration"
 version: 0.1.0
-tag: getporter/porter-hello
+registry: getporter
+reference: getporter/hello:v0.1.0
```

Comment:
Why are we setting both in this test file?

Reply:
Hmmm good Q.  Must be a mistake.  Will remove.